activiti-core/activiti-engine/src/main/java/org/activiti/engine/repository/ProcessDefinitionQuery.java

```diff
@@ -140,10 +141,16 @@ public interface ProcessDefinitionQuery extends Query<ProcessDefinitionQuery, Pr
     ProcessDefinitionQuery processDefinitionResourceNameLike(String resourceNameLike);
 
     /**
-     * Only selects process definitions which given userId is authoriezed to start
+     * Only selects process definitions which given userId is authorized to start
      */
     ProcessDefinitionQuery startableByUser(String userId);
 
+    /**
+     * Only selects process definitions which given group members are authorized to start
+     * If not set and startableByUser is set, the members of that user will be used
```

Comment:
I don't understand  what this means: `the members of that user will be used` can you explain?

Reply:
it should be `the groups of that user` :) I will update it